duo_openvpn_as.py

```diff
@@ -512,6 +510,8 @@ def connect(self):
                                 certfile=self.cert_file,
                                 cert_reqs=self.cert_reqs,
                                 ca_certs=self.ca_certs)
+    ssl_version_blacklist = ssl.OP_NO_SSLv2 | ssl.OP_NO_SSLv3
+    self.sock._context.options |= ssl_version_blacklist
```

Comment:
My understanding is that the wrap_socket based approach is deprecated?  see https://github.com/duosecurity/duo_openvpn/commit/3f0b9383b4717e63946e4fbb2daec7c8033bfcf8 for how this was approached in the openvpn repo.

Reply:
oh nice! I'll pivot to that approach real quick